packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -412,8 +411,7 @@ function FindUsername({ username, onConnect, onDone, onClose, onUsernameChange =
                                 <IconButton
                                     className={findUsernameClasses.button}
                                     // @ts-expect-error https://github.com/mui-org/material-ui/pull/26064
-                                    color={username ? 'primary' : 'default'}
-                                    disabled={!username}>
```

Comment:
I think this is required, right? The button should not react to clicks if the username absent.